### ISSUE TITLE
Train/Val Reports on wandb can not be opened

### ISSUE THREAD

**Issue by Assioncreed** [wgcban/ddpm-cd#38]
Excuse me, I would like to know how much time it took to train the change detection model because Train/Val Reports on wandb won't open。When I trained on my own, it took a long time but the progress still didn't move

**Comment by wgcban** [comment]
@Assioncreed Thanks. Yes, due to space limit I have to delete them. However, you can still access the pre-trained models.

**Comment by Assioncreed** [comment]
@wgcban Thank you for your reply, do you remember how long it took to train the CD model?I'm training myself, but I'm not making progress for a long time

**Comment by wgcban** [comment]
@Assioncreed are you fine-tuning the CD head? It should not take longer time. Should be around ~1 day maximum. Make sure everything is going right by checking loss curves, etc.

**Comment by Assioncreed** [comment]
@wgcban OK, maybe something went wrong, thank you again
